modules/c3-assisted-installer-preparing-image-generating.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-assisted-installer-preparing-image-generating_{context}"]
+= Generating the image in the {ai-full}
+
+Create a cluster and download the discovery ISO image.
+
+.Procedure
+
+. Log in to link:https://console.redhat.com/[{ai-full} web console] with your credentials.
+
+. In the *Red Hat OpenShift* tile, select *OpenShift*. 
+
+. In the *Red Hat {product-title}* tile, select *Create Cluster*.
+
+. On the *Cluster Type* page, scroll to the end of the *Cloud* tab, and select *Oracle Cloud Infrastructure (virtual machines)*. 
+
+. On the *Create an OpenShift Cluster* page, select the *Interactive* tile.
+
+. On the *Cluster Details* page, complete the following fields:
++
+[cols="1,3",options="header",subs="quotes"]
+|===
+|Field |Action required
+
+|*Cluster name*
+|Specify the name of your {product-title} cluster. This name is the same name you used to create the resource via the Terraform scripts. The name must be between 1-54 characters. It can use lowercase alphanumeric characters or hyphen (-), but must start and end with a lowercase letter or a number.
+
+|*Base domain*
+|Specify the base domain of the cluster. This is the value used for the `zone_dns` variables in the Terraform scripts that run on {oci-c3-short}. Make a note of the value.
+
+|*OpenShift version*
+| Select *OpenShift 4.16.20*. If it is not immediately visible, scroll to the end of the dropdown menu, select *Show all available versions*, and type the version in the search box.
+
+|*Integrate with external partner platforms*
+|Select *Oracle Cloud Infrastructure*.
+
+After you specify this value, the *Include custom manifests* checkbox is selected by default and the *Custom manifests* page is added to the wizard.
+|===
+
+. Leave the default settings for the remaining fields, and click *Next*.
+
+. On the *Operators* page, click *Next*.
+
+. On the *Host Discovery* page, click *Add hosts* and complete the following steps:
++
+[NOTE]
+====
+The minimal ISO image is the mandatory *Provisioning type* for the {oci-first-no-rt}, and cannot be changed.
+====
+
+.. In the *SSH public key* field, add the SSH public key by copying the output of the following command: 
```

Comment:
This is just something to consider and probably fine to leave as is if your other Oracle procedures are worded this way but - I feel like, especially considering the [docs guideline section](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command) about explicitly using wording like "by running the following command", this might be _just enough_ actions to merit two separate steps. 

How about breaking this up into two steps like "Create the SSH public key by running the following command:" and then "In the *SSH public key* field, add the SSH public key by entering the output of the previous step's command."?